raiden-dapp/tests/unit/components/transfer/transfer-inputs.spec.ts

```diff
@@ -0,0 +1,65 @@
+jest.mock('vue-router');
+import Mocked = jest.Mocked;
+import { mockInput } from '../../utils/interaction-utils';
+import { mount, Wrapper } from '@vue/test-utils';
+import Vue from 'vue';
+import store from '@/store';
+import VueRouter from 'vue-router';
+import { RouteNames } from '@/router/route-names';
+import Vuetify from 'vuetify';
+import TransferInputs from '@/components/transfer/TransferInputs.vue';
+import { TestData } from '../../data/mock-data';
+import { One } from 'ethers/constants';
+import { generateToken } from '../../utils/data-generator';
+
+Vue.use(Vuetify);
+
+describe('TransferInputs.vue', () => {
+  const vuetify = new Vuetify();
+  const router = new VueRouter() as Mocked<VueRouter>;
+  const token = generateToken();
+
+  router.push = jest.fn().mockImplementation(() => Promise.resolve());
+
+  const wrapper: Wrapper<TransferInputs> = mount(TransferInputs, {
+    vuetify,
+    store,
```

Comment:
The store isn't used. Why injecting it here? Is this necessary? Else mock it as empty object.

Reply:
It is, tests throws an error if the store is not included.

Reply:
See the [comment](https://github.com/raiden-network/light-client/pull/2055#discussion_r469896026) above.